src/nonblocking/mod.rs

```diff
@@ -0,0 +1,2 @@
+#![cfg(feature = "nonblocking")]
+pub mod local_socket;
```

Comment:
```suggestion
pub mod local_socket;
```
(As per `lib.rs` suggestion, you no longer need the inner attribute here if you put it in `lib.rs`.)